nixos/modules/services/misc/gollum.nix

```diff
@@ -0,0 +1,96 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.gollum;
+in
+
+{
+  options.services.gollum = {
+    enable = mkOption {
+      type = types.bool;
+      default = false;
+      description = "Enable the Gollum service.";
+    };
+
+    package = mkOption {
```

Comment:
Is there a use case for using a package other than the default?

Reply:
Nope, I just saw this pattern in other packages. Will remove it.